starport/pkg/cosmosaccount/cosmosaccount.go

```diff
@@ -0,0 +1,216 @@
+package cosmosaccount
+
+import (
+	"errors"
+	"os"
+
+	dkeyring "github.com/99designs/keyring"
+	"github.com/cosmos/cosmos-sdk/crypto/hd"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/go-bip39"
+)
+
+// KeyringServiceName used for the name of keyring in OS backend.
+const KeyringServiceName = "starport"
+
+// KeyringHome used to store account related data.
+var KeyringHome = os.ExpandEnv("$HOME/.starport/accounts")
+
+var (
+	ErrAccountExists       = errors.New("account already exists")
+	ErrAccountDoesNotExist = errors.New("account does not exist")
+)
+
+const (
+	accountPrefixCosmos = "cosmos"
+	pubKeyPrefix        = "pub"
+)
+
+// Registry for accounts.
+type Registry struct {
+	kr keyring.Keyring
+}
+
+// New creates a new registry to manage accounts.
+func New(backend string) (Registry, error) {
+	kr, err := keyring.New(KeyringServiceName, backend, KeyringHome, os.Stdin)
+	if err != nil {
+		return Registry{}, err
+	}
+
+	r := Registry{
+		kr: kr,
+	}
+
+	return r, nil
+}
+
+// Account represents an CosmosSDK account.
+type Account struct {
+	// Name of the account.
+	Name string
+
+	// Info holds additional info about the account.
+	Info keyring.Info
+}
+
+// Address returns an address for acocunt for given prefix.
```

Comment:
```suggestion
// Address returns the address of the account from given prefix.
```

Reply:
there can be different addresses for a single account for different prefixes

Reply:
Yes but there is a single one for a given prefix?

Reply:
ah, yes

Reply:
Let at least fix the type on `acocunt`